service/policy/attributes/attributes.proto

```diff
@@ -86,10 +86,17 @@ message ListAttributesResponse {
 }
 
 message GetAttributeRequest {
-  // Required
+  // Deprecated
   string id = 1 [
```

Comment:
I'd like to discuss this breaking change and the pros/cons. An alternative could be allowing either `id` or `fqn` without `oneof`, then logically preferring `id` in the event both were provided.

Reply:
This was discussed with @jakedoublev and @jrschumacher with a decision to move forward with the use of oneof